### ISSUE TITLE
Can't import username

### ISSUE THREAD

**Issue by fred01** [xbreathoflife/hsqldb-csv-importer#1]
Hi!
Thanks a lot for this awesome project. But I experienced an error when I tried to import a file with the following content:
```
Username,Identifier,First name,Last name
booker12,9012,Rachel,Booker
grey07,2070,Laura,Grey
johnson81,4081,Craig,Johnson
jenkins46,9346,Mary,Jenkins
smith79,5079,Jamie,Smith
sarac,90210,Sarah,O'Connor
```
Can you, please, help with that?

**Comment by fred01** [comment]
Would you please create PR to fix this issue?

**Comment by xbreathoflife** [comment]
> Would you please create PR to fix this issue?

Hi! I have fixed this in f7cac300623028758f7f487bbcd5b95d5fc5671c
